include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -75,34 +75,60 @@ struct __subgroup_radix_sort
         }
     };
 
-    template <typename _KeyT, typename _Wi, typename _Src, typename _Keys>
+    template <typename _ValT, bool __uniform_space, typename _Wi, typename _Src, typename _Vals>
     static void
-    __block_load(const _Wi __wi, const _Src& __src, _Keys& __keys, const uint32_t __n, const _KeyT& __default_key)
+    __block_load(const _Wi __wi, const _Src& __src, _Vals& __vals, const uint32_t __n)
```

Comment:
a lot of missing `::std::` before primitive types (like `uint32_t`)?

Reply:
missing `::std::` also was before the changes... Probably, it makes sense to "clean up" that as a separate changes for non increasing number of non-relevant changes in that PR?